config.json.dist

```diff
@@ -58,7 +61,8 @@
     "includeOrganizerLabel": "",
     "countries": [
       {"code": "BE", "default": true},
-      {"code": "NL", "default": false}
+      {"code": "NL", "default": false},
+      {"code": "BOOKABLE_EVENT", "default": false}
```

Comment:
I think we should leverage the standards defined in ISO-3166-1993 here and use `ZZ` as country code to indicate an `Unknown or unspecified country`

Reply:
Good Idea, do you think it will be clear enough?
Since these codes are also used for the translations of labels:
[https://github.com/cultuurnet/udb3-angular/blob/0710d3350054a215b39dd8bc06ceabb5d03a95f9/src/core/translations/dutch-translations.constant.js#L16](https://github.com/cultuurnet/udb3-angular/blob/0710d3350054a215b39dd8bc06ceabb5d03a95f9/src/core/translations/dutch-translations.constant.js#L16)

Reply:
I think it will be clearer than the current situation where we add `BOOKABLE_EVENT` to a list of country codes